Libplanet.Net/Consensus/Gossip.cs

```diff
@@ -401,6 +433,14 @@ private async Task CheckValidatorsLiveness(CancellationToken ctx)
         {
             while (!ctx.IsCancellationRequested)
             {
+                if (!_table.Any())
```

Comment:
IMO this condition is not very powerful. New block can be committed only when 2/3 of the validator power exists in the network, but `_table` at this moment is a list of peers given by `--consensus-peer`s in Headless argument.

Reply:
For the weak condition, the `MinimumPeerCount` is added. If the `countOfPong >= MinimumPeerCount`, then consensus `Gossip` would not proceed (blocking). The value of `MinimumPeerCount` is chosen 4, which is the least value of BFT (`where f = 1`).